Dockerfile.dev

```diff
@@ -18,6 +18,10 @@ ENV PYTHONPATH /app/
 ENV FLASK_APP=datastore.$MODULE.app
 ENV FLASK_ENV=development
 
+RUN mkdir -p /run/secrets
+RUN echo -n openslides >/run/secrets/datastore_postgres_user
+RUN echo -n openslides >/run/secrets/datastore_postgres_password
+
```

Comment:
If you want to follow @jsangmeister's suggestion, then delete these lines and follow the other comment [1].

Reply:
Okay, these code lines are a workaround, so using a constant in DEV_Mode is better.